src/plays/tic-tac-toe/TicTacToe.jsx

```diff
@@ -0,0 +1,28 @@
+import { getPlayById } from "meta/play-meta-util";
+
+import PlayHeader from "common/playlists/PlayHeader";
+
+import Game from "./Game";
+import "./TicTacToe.css";
+
+function TicTacToe(props) {
+  // Do not remove the below lines.
+  // The following code is to fetch the current play from the URL
+  const { id } = props;
+  const play = getPlayById(id);
+
+  // Your Code Start below.
+
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={play} />
+        <div className="play-details-body">
```

Comment:
Add a unique class name here and use that to prefix all the styles so that no chance of any conflicts.

Reply:
I can move `tictactoe.css` to `Game.jsx` and it would work the same. Can I do that?


Or, are you suggesting that I need to rename every CSS class from the entire play with a unique prefix? 
Like, `ttt_buttonGroup`, `ttt_icon` etc?